x-pack/plugins/security_solution/common/detection_engine/schemas/response/rules_schema.test.ts

```diff
@@ -764,7 +764,7 @@ describe('rules_schema', () => {
 
     test('should return 5 fields for a rule of type "threat_match"', () => {
```

Comment:
Should these two numbers match?

```suggestion
    test('should return 8 fields for a rule of type "threat_match"', () => {
```